db/migrate/20231010180442_create_mentee_questionnaires.rb

```diff
@@ -0,0 +1,25 @@
+class CreateMenteeQuestionnaires < ActiveRecord::Migration[7.0]
+  def change
+    create_table "mentee_questionnaires", id: :binary, force: :cascade do |t|
+      t.binary "respondent_id", limit: 16, null: false, foreign_key: true
+      t.string :name, null: false
+      t.string :country, null: false
+      t.string :city, null: false
+      t.string :work_url, null: false
```

Comment:
```suggestion
      t.string :work_url
```

They might not be working yet.